infra/tf/better_uptime/vars.tf

```diff
@@ -0,0 +1,25 @@
+# MARK: Better Uptime
+variable "better_uptime_monitors" {
+	type = list(object({
+		url = string
+		public_name = string
+	}))
+}
+
+variable "better_uptime" {
+	type = object({
+		// The name of your company. This will be displayed on your status page
+		// in the top left. This is required by Better Uptime.
+		company_name = string
+		// The URL of your company. This will be used on the status page to link
+		// to your company's website. This is required by Better Uptime.
+		company_url = string
+		// The subdomain is the part of the public URL of your status page uses.
+		//
+		// Eg. <company_subdomain>.betteruptime.com.
+		//
+		// It needs to be unique across all of Better Uptime. This is required
+		// by Better Uptime.
+		company_subdomain = string
```

Comment:
Nit:

These docs would be more helpful in the ns.rs instead of here. 

Putting it here is pretty much reiterating what the BU TF docs already say, and it's pretty clear where it's getting passed.

Putting it in the ns.rs is much more helpful since I don't want to go dig through Bolt and TF to see where this gets passed.